l1-contracts/contracts/state-transition/ValidatorTimelock.sol

```diff
@@ -180,15 +181,19 @@ contract ValidatorTimelock is IExecutor, Ownable2Step {
 
     /// @dev Check that batches were committed at least X time ago and
     /// make a call to the hyperchain diamond contract with the same calldata.
-    function executeBatches(StoredBatchInfo[] calldata _newBatchesData) external onlyValidator(ERA_CHAIN_ID) {
-        _executeBatchesInner(ERA_CHAIN_ID, _newBatchesData);
+    function executeBatches(
```

Comment:
ValidatorTimelock should be backward compatible with all chain version from now and later, so I propose adding function `executeBatches(StoredBatchInfo[] calldata _newBatchesData)` so chains that haven't update yet would work as expected.

Better to make the one interface to not redeploy contract in the future at all, but let discuss it